docs/blog/newsletter-november-2022.rst

```diff
@@ -0,0 +1,131 @@
+.. post:: November 07, 2022
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – November 2022
+#########################################
+
+Hello once again from the newsletter team. We have a lot of things going on in the community right now.
+
+I wrote last month about opportunities to be a part of the Write the Docs community, and this month sees more information. A `new Write the Docs Enhancement Proposal <https://github.com/writethedocs/weps/pull/6>`__ has been opened to provide more clarity on Write the Docs teams and what they do. The change also includes Mikey Ariel stepping down as Prague conference chair and starting as Community Board chair. While I'm sad to see an end to an era of conferences very near me, I'm excited for what she'll bring in the newly defined role. In related news, we have the dates for `conferences in 2023 </blog/2023-conference-announcements/>`__ that you'll probably want to save.
```

Comment:
Strictly speaking, this is about being more involved, not about being part of the community (arguably by reading you're already part, what do you think?

Reply:
Also arguably, it's not an end to the Prague conferences, just an announcement that one is not happening in 2023.

Reply:
Good points, both of them.